kujira/app/components/charts/bar-chart.js

```diff
@@ -0,0 +1,32 @@
+import Ember from 'ember';
+import {
+    createRefreshingChart
+} from './chartsCommonContent';
+
+export default Ember.Component.extend({
+    height: null,
+    width: null,
+    chartDescription: null,
+    dataType: null,
+    refreshingInterval: null,
+
+    willDestroyElement() {
+        clearInterval(this.get('refreshingInterval'));
+    },
+
+    didRender() {
+
+        var dataType = this.get('dataType'),
+            div = this.get('element'),
+            width = this.get('width'),
+            height = this.get('height'),
+            chartDescription = this.get('chartDescription'),
+            refreshPeriod = 5000,
+            chartType = 'barChart',
+            chartData;
```

Comment:
You have declared a variable (at this point it is 'undefined'), just to pass it later on to function. This problem occurs in every chart.